tasks/extract-binaries-from-image/tests/mocks.sh

```diff
@@ -0,0 +1,40 @@
+# mocks to be injected into task step scripts
+
+set -x
+
+function skopeo() {
+  echo "Mock skopeo called with: $*"
+  echo "$*" >> $(workspaces.data.path)/mock_skopeo.txt
+
+  if [[ "$*" != "copy docker://registry.io/image:tag dir:/"* ]]
+  then
+    echo Error: Unexpected call
+    exit 1
+  fi
+
+  cp $(workspaces.data.path)/image_data/* $TMP_DIR/
+
+#   cat > $TMP_DIR/manifest.json << EOF
```

Comment:
Can you remove the commented code? I see the tests passed, so I am guessing it isn't needed

Reply:
Done!